jekyll/_data/sidenavstyle.yml

```diff
@@ -0,0 +1,22 @@
+en:
+- name: Style
+  icon: icons/sidebar/getting_started.svg
+  children:
+    - name: CircleCI Headings
+      link: style/style-headings/
+    - name: CircleCI Links
+      link: style/style-links/
+    - name: CircleCI Overview
+      link: style/style-overview/
+
+ja:
+- name: Style
+  i18n_name: はじめよう
+  icon: icons/sidebar/getting_started.svg
+  children:
+    - name: Overview
+      i18n_name: 概要
+      children:
+        - name: Welcome
+          i18n_name: ようこそ
+          link: ja/style/style-overview/
```

Comment:
Are we sure this is correct? this doesn't match the data set in `en`?

Reply:
This is just temporary data in JA the JA pages are not made yet nor is the content complete. You have a good point though I can make the place holders match :)